packages/coachmark/src/spectrum-config.js

```diff
@@ -13,62 +13,56 @@ governing permissions and limitations under the License.
 
 import { converterFor } from '../../../tasks/process-spectrum-utils.js';
 
-const converter = converterFor('spectrum-CoachMarkIndicator');
+const converter = converterFor('spectrum-CoachMark');
 
 /**
  * @type { import('../../../tasks/spectrum-css-converter').SpectrumCSSConverter }
  */
 const config = {
     conversions: [
         {
-            inPackage: '@spectrum-css/coachmark',
+            inPackage: '@spectrum-css/coachindicator',
             outPackage: 'coachmark',
-            fileName: 'coachmark',
-            excludeByComponents: [
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-image',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-content',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-footer',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-header',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-step',
-                },
-                {
-                    type: 'class',
-                    name: 'spectrum-CoachMarkPopover-title',
-                },
-            ],
+            fileName: 'coach-indicator',
             components: [
-                converter.classToHost(),
-                converter.classToClass('spectrum-CoachMarkIndicator-ring'),
+                converter.classToHost('spectrum-CoachIndicator'),
+                converter.classToClass('spectrum-CoachIndicator-ring', 'ring'),
                 converter.classToAttribute(
-                    'spectrum-CoachMarkIndicator--quiet'
+                    'spectrum-CoachIndicator--quiet',
+                    'quiet'
                 ),
                 ...converter.enumerateAttributes(
                     [
-                        ['spectrum-CoachMarkIndicator--dark'],
-                        ['spectrum-CoachMarkIndicator--light'],
+                        ['spectrum-CoachIndicator--dark', 'dark'],
+                        ['spectrum-CoachIndicator--light', 'light'],
                     ],
                     'variant'
                 ),
             ],
         },
+        {
+            inPackage: '@spectrum-css/coachmark',
+            outPackage: 'coachmark',
+            fileName: 'coachmark',
+            components: [
+                converter.classToHost(),
```

Comment:
Where are the spectrum-CoachMarkPopover styles being mapped to for the web component?

Reply:
the styles are referenced as spectrum-coachmark rather than spectrum-CoachMarkPopover